src/plugin/PluginBase.js

```diff
@@ -438,10 +451,20 @@ define([
 
         commitMessage = message ? commitMessage + ' - ' + message : commitMessage;
 
+        if (this.callDepth > 0) {
+            self.logger.debug('Call-depth is greater than zero, will not persist "', this.callDepth, '"');
+            // How to handle this neatly..
+            self.addCommitToResult(commitMessage);
```

Comment:
It would have worked, but I changed it to call `self.result.addCommitMessage` right away